Assets/Mirage/Editor/WelcomeWindow/Resources/WelcomeWindow.uss

```diff
@@ -108,17 +108,17 @@ Button {
 }
 
 .dark-selected-tab {
-    background-color: rgba(53, 53, 53, 255);
```

Comment:
I don't think this has anything to do with changelog.  Should be a separate pr

Reply:
Didn't see that. I think Unity automatically changed it for some reason. Will remove this and move it to a different PR later.